lib/Rex/Test/Base/has_checksum.pm

```diff
@@ -0,0 +1,54 @@
+#
+# (c) Jan Gehring <jan.gehring@gmail.com>
```

Comment:
IANAL, but I'm pretty sure in this case the copyright is being held by the author of this file (and not by Jan :)).